src/app/components/footer/footer.component.html

```diff
@@ -1,5 +1,14 @@
 <footer class="tui-container tui-container_fullwidth tui-space_top-6 footer">
-    <p class="tui-island__paragraph tui-space_vertical-0">2022 &copy; Course Gamification</p>
+    <p class="tui-island__paragraph tui-space_vertical-0">
+        2022 &copy; Course Gamification is licensed under CC BY-NC-SA 4.0
+        <a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/">
+            <img alt="Creative Commons License"
+                 [src]="isNightMode()? 'assets/global/cc%20by-nc-sa%20dark%20mode.png' : 'assets/global/cc%20by-nc-sa.png'"
```

Comment:
Probably if you change this to `headerComponent.isNightMode()` it will work without refresh.